### ISSUE TITLE
UNC Paths not respected with Windows 10 v20H2 and NP++ v793 x64.

### ISSUE THREAD

**Issue by sky5walk** [notepad-plus-plus/notepad-plus-plus#9670]
I see a similar problem now with Windows 10 v20H2 and NP++ v793 x64.

File-Open-dialog and paste in --> \\1.2.3.4\blah\more\evenmore\myfile.txt
The dialog complains after truncating one leading backslash? --> \1.2.3.4\blah\more\evenmore\myfile.txt

[Window Title] Open
[Content]
\1.2.3.4\blah\more\evenmore\myfile.txt
File does not exist.
Check the path and try again.
[OK]

_Originally posted by @sky5walk in https://github.com/notepad-plus-plus/notepad-plus-plus/issues/7772#issuecomment-802283427_

**Comment by sasumner** [comment]
Similar/duplicate of #9582 which is scheduled to be fixed with Notepad++ 7.9.5.
Please check results obtained with 7.9.5 and report back in 9582 if any problems.

**Comment by mere-human** [comment]
@sky5walk Does it work in 7.9.4?

**Comment by sky5walk** [comment]
Is v795 days away or weeks?

**Comment by mere-human** [comment]
> Is v795 days away or weeks?

Who knows. You can ask Don. Why does it matter for this issue?
Can you try in Notepad++ 7.9.4 portable?

**Comment by sky5walk** [comment]
Because I use both "\\somepath" and \\somepath.
And v795 supposedly fixes 1 or both?
I will try v794 when time available today.

**Comment by sasumner** [comment]
>Because I use both `"\\somepath"` and `\\somepath`

`\\somepath` should work in 7.9.4

`"\\somepath"` should work in 7.9.5, when released (actually "both" types should work)

So your obvious choice are:

* try 7.9.4 and (hopefully) have success with `\\somepath`
* try 7.9.4 and, when you find yourself using `\\somepath"`, simply remove the `"` characters before continuing with the action
* use a version 7.9.2 or earlier

**Comment by sky5walk** [comment]
Confirmed v794 works for un-double-quoted unc paths but not double-quoted paths like those delivered by Windows File Explorer - RightMouse Click - Copy as path.
Waiting on v795...
